examples/server.js

```diff
@@ -23,7 +23,7 @@ function listDirs(root) {
   return dirs;
 }
 
-function getIndexTemplate() {
+function getIndexTemplate ( ){
```

Comment:
format: style mismatch:
```suggestion
function getIndexTemplate() {
```

`2f4f64f1` ⏎⏎ at column 2 should be removed.
`16b835cf` ␣ at column 26 should be removed.
`16b835cf` ␣ at column 28 should be removed.
`39615007` ␣ should be inserted at column 30.